src/app/common/interfaces/mainflux.interface.ts

```diff
@@ -80,3 +80,16 @@ export interface MainfluxMsg {
   sum: number;
   protocol: string;
 }
+
+export interface MsgFilters {
+  subtopic?: string;
+  offset?: number;
+  limit?: number;
+  from?: number;
+  to?: number;
+}
+
+export interface DateFilter {
```

Comment:
Since you use this filter only in message-table.component.ts, maybe a simple unexported typedef there would be enough. No need to make it globally available.

Reply:
After my last commit this is not true anymore. Let's keep this interface.